source/jquery.flot.time.js

```diff
@@ -142,7 +142,7 @@ API.txt for details.
 					case 'q':
 						c = "" + (Math.floor(d.getMonth() / 3) + 1); break;
 					case 'S': c = leftPad(d.getSeconds()); break;
-					case 's': c = "" + formatMicroseconds(d.getMicroseconds(),decimals); break;
+					case 's': c = "" + formatSubSeconds(d.getMilliseconds(), d.getMicroseconds(), decimals); break;
```

Comment:
This change is necessary since `microsecondDate.getMicroseconds()` no longer returns whole sub-second value. I also cleaned up the `formatMicroseconds()` method (renamed to `formatSubSeconds()`)